packages/features/schedules/lib/use-schedule/useSchedule.ts

```diff
@@ -35,6 +37,7 @@ export const useSchedule = ({
       endTime: (prefetchNextMonth ? nextMonthDayjs : monthDayjs).endOf("month").toISOString(),
       timeZone: timezone!,
       eventTypeId: eventId!,
+      duration: duration ? `${duration}` : undefined,
```

Comment:
`useSchedule` hook didn't event take duration into account yet. So this was definitely a bug!